src/rest/request.cpp

```diff
@@ -105,6 +89,13 @@ std::string Request::GetHeaderValue(const std::string aHeaderField) const
     return (it == mHeaders.end()) ? "" : it->second;
 }
 
+std::string Request::GetQueryValue(const std::string aQueryName) const
+{
+    auto it = mHeaders.find(aQueryName);
+
+    return (it == mHeaders.end()) ? "" : it->second;
```

Comment:
Shouldn't we use `mQueryParameters` here?

Reply:
Yes we definitely should.
I've tested the parsing code but not the api. Looking at the automated tests its gonna be hard to test without having a endpoint that actually uses the query parameters. We have code under works that uses them but only for timeouts so that would not be testable either.